ios/Podfile

```diff
@@ -64,6 +64,9 @@ abstract_target 'Status' do
   pod "Keycard", git: "https://github.com/status-im/Keycard.swift.git", tag: '3.1.1'
   pod 'secp256k1', git: "https://github.com/status-im/secp256k1.swift.git", submodules: true
 
+  pod 'FirebaseCore', :modular_headers => true
+  pod 'GoogleUtilities', :modular_headers => true
+
```

Comment:
Here we're configuring the iOS build to resolve Firebase and Google dependencies with `modular_headers`. Without this change the build will fail because these dependencies do not export Swift modules (I think).